_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_ddi.cpp

```diff
@@ -83,58 +83,58 @@ DriverEncoder* CreatePlatformH265Encoder(VideoCORE* core, ENCODER_TYPE type)
 }
 
 // this function is aimed to workaround all CAPS reporting problems in mainline driver
-mfxStatus HardcodeCaps(ENCODE_CAPS_HEVC& caps, VideoCORE* core)
+mfxStatus HardcodeCaps(MFX_ENCODE_CAPS_HEVC& caps, VideoCORE* core)
```

Comment:
Do we need to use new type everywhere? This specific function looks like good candidate for leaving just ENCODE_CAPS_HEVC.

Reply:
I agree with next changes (and mb FixDirtyRect), but in this function we can hardcode caps which doesnt report by driver, including new checks.